CHANGELOG.md

```diff
@@ -2,7 +2,69 @@
 
 All notable changes to this project will be documented in this file.
 
-## Unreleased
+## 1.0.0
+
+### :boom: Breaking Change
```

Comment:
need to add breaking changes/upgrade guidelines for otlp exporter changes

Reply:
They are there

Reply:
OH yeah sorry i don't why i expected it to be in this PR